rules/windows/defense_evasion_whitespace_padding_in_command_line.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2021/07/30"
+maturity = "production"
+updated_date = "2021/07/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution events where the command line value contains superfluous whitespace characters. Attackers
```

I think this is more accurate and a little stronger
```suggestion
Identifies process execution events where the command line value contains a long sequence of whitespace characters. Attackers
```